profiler/data.py

```diff
@@ -70,45 +82,50 @@ def close(self):
 
 class FileBasedProfileDB(ProfileDB):
     """Represents a file-based implementation of a ProfileDB
-    runs database. Each run is stored as a separate file under a subdirectory structured as `<process_name>/<timestamp>`.
+    runs database. Each run is stored as a separate file under a subdirectory structured as `<command>/<timestamp>`.
     """
 
-    def __init__(self, path: str = PROFILE_PATH):
+    def __init__(self, path: str = DEFAULT_PROFILE_DB_PATH):
         self.path = path
         if not os.path.exists(self.path):
             os.mkdir(self.path)
 
     def __str__(self):
         result = ""
         if os.path.exists(self.path):
-            for process in os.listdir(self.path):
+            for command_hash in glob.glob(self.path + "/*"):
+                with open(os.path.join(command_hash, "command.txt"), "r") as f:
+                    command = f.read()
+                n_runs = len(glob.glob(os.path.join(command_hash, "*.json")))
                 result += (
-                    f"{process}: "
-                    + str(len(os.listdir(f"{self.path}/{process}")))
-                    + "\n"
+                    f"[{command_hash.split('/')[-1]}] \"{command}\": {n_runs} runs\n"
                 )
             return result
         return ""
 
-    def find(self, process) -> List[ProfileData]:
-        key = improve_profileDB_key(process)
+    def find(self, command) -> List[ProfileData]:
```

Comment:
Is command the process plus its arguments?

Reply:
yes, it's the full command, which includes the args